README.md

```diff
@@ -4,64 +4,170 @@ Rubyric is an online tool in which you can create courses and assignments,
 receive submissions and assess them by using rubrics. More information about 
 Rubyric and how to use it can be found [here](doc/rubyric.md).
 
-## Install environment
-Install ruby version 2.3.1 or higher, for example with rvm or rbenv. Then
-install rails:
+## Environment
+Install ruby 2.3.1 or higher. One way of installing these versions in Linux is 
+by using one of the following Ruby version management tools, 
+[rvm or rbenv](https://linuxize.com/post/how-to-install-ruby-on-ubuntu-20-04/). 
+
+If you followed the guides provided in the above link. You will find yourself 
+adding some scripts  to the `~/.bashrc` file. However, you can use your favourite
+shell.
+
+You can always verify your default shell by typing the following command in your 
+terminal
+
+```sh 
+ls -l /proc/$$/exe
```

Comment:
`echo $SHELL` should typically be enough, but that works too on Linuxes (assuming procfs is mounted).

Reply:
I will add that option too.